General report updates - interpreter/compiler usage fixes and code snippets (#81)
chhoumann/p4: Report/sections/3_implementation/antlr.tex

\section{ANTLR Parsing Tool}\label{sec:ANTLRTool}
In order to parse a language using a grammar, one must decide whether to manually write the parser or use a tool to generate it\cite{crafting_a_compiler}.

> Based on the fact that writing a parser is a rather repetitious and mechanical task, we followed our professor's advice and decided to use the parser generator \textit{ANTLR} for our language, which was recommended in the Languages and Compilers course. 

Reviewer: ```suggestion
Writing a parser is a rather repetitious and mechanical task. Therefore, we followed our professor's advice and decided to use the parser generator \textit{ANTLR} for our language, as recommended in the Languages and Compilers course. 
```
